app/src/js/withUrlHelper.js

```diff
@@ -0,0 +1,87 @@
+/* eslint-disable import/no-cycle */
```

Comment:
I don't think we need this, do we? There shouldn't be cyclic dependencies if this is a module we're creating and we've only imported third-party modules?

Reply:
I will test again and see.

Reply:
Removed. Tested. Looks good. Change committed.